react-framework-master/server/schemaSingle.js

```diff
@@ -0,0 +1,55 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/restaurants');
+var faker = require('faker');
+// var db = mongoose.connection;
+
+var Restaurant = mongoose.model('Restaurant', {
+	restaurant_id: Number,
+	street_address: String,
+	cross_street: String,
+	neighborhood: String,
+	brunch_hrs: String,
+	lunch_hrs: String,
+	dinner_hrs: String,
+	food_type: String,
+	dining_style: String,
+	dress_code: String,
+	payment_options: String,
+	chef: String,
+	website: String,
+	phone_number: String,
+	longitude: Number,
+	latitude: Number
+});
+
+// for(var i=0; i< 100; i++) {
```

Comment:
Think I commented on this already, but are these commented out lines necessary?

Reply:
You are right will clean out anything thats commented